sources/appendix-value-extraction.asciidoc

```diff
@@ -256,18 +256,26 @@ TODO
 
 ==== Plugging in custom extractors
 
-Additional value extractors can be registered with the Bean Validation provider in the following ways:
-
-* Invoke the new method `Configuration#addValueExtractor(ValueExtractor<?>)` (to apply it at the validation factory level)
-* Invoke the new method `ValidatorContext#addValueExtractor(ValueExtractor<?>)` (to apply it for a single `Validator`)
-* Specify the fully qualified class name of one or several extractors in `META-INF/validation.xml`:
+Additional value extractors can be plugged in when bootstrapping a `Validator` or `ValidatorFactory`, amending and/or overriding the set of built-in value extractors.
+The following ways to do so exist, in descending order of precedence:
 
+* Invoking the new method `ValidatorContext#addValueExtractor(ValueExtractor<?>)` (to apply it for a single `Validator`)
```

Comment:
Really nitpicking but I don't think we should use `new` in the spec. I would simply remove the word here and in the line below.